db/noop.go

```diff
@@ -0,0 +1,19 @@
+package db
+
+// NoopDB implements the DB interface with Noops
+type NoopDB int
+
+// Init noop
+func (n *NoopDB) Init(c *Config) (AuthDB, error) {
+	return n, nil
+}
+
+// IsRevoked noop
+func (n *NoopDB) IsRevoked(sn string) (bool, error) {
+	return false, nil
+}
+
+// Revoke noop
+func (n *NoopDB) Revoke(rci *RevokedCertificateInfo) error {
+	return nil
```

Comment:
we can return an ErrNotImplemented error

Reply:
if we're going to return that for Revoke then shouldn't we returning that for IsRevoked as well? Sort of stops being a NoopDB then though.

Reply:
... well, the cert is not revoked if NoopDB is used, but it cannot revoke.